tests/shared/importers/test_importer.py

```diff
@@ -368,15 +367,6 @@ def test_nlu_data_domain_sync_responses(project: Text):
     assert "utter_rasa" in domain.responses.keys()
 
 
-def test_importer_with_invalid_model_config(tmp_path: Path):
-    invalid = {"version": "2.0", "policies": ["name"]}
-    config_file = tmp_path / "config.yml"
-    rasa.shared.utils.io.write_yaml(invalid, config_file)
-
-    with pytest.raises(YamlValidationException):
-        TrainingDataImporter.load_from_config(str(config_file))
```

Comment:
This test moved to `tests/graph_components/validators/test_default_recipe_validator.py` as auto-config is dependent on recipe and not importer.